kolibri/core/assets/src/asset_loader.js

```diff
@@ -9,7 +9,7 @@
 const scriptjs = require('scriptjs');
 const loadcss = require('fg-loadcss').loadCSS;
 const onloadcss = require('fg-loadcss/src/onloadCSS');
-const logging = require('loglevel');
+const logging = require('./logging').getLogger(__filename);
```

Comment:
Can get the logger in the same way.

Outside of the core app, you should still be able to `require('loglevel')` and get the new logging interface.